client/src/components/kvEditor/KVEditor.tsx

```diff
@@ -27,13 +36,17 @@ function KVEditor({ name, kvs, setKvs, readOnly, hasEnvSupport, env }: KVEditorP
     return result;
   }, [kvs, readOnly]);
 
-  const onChangeRowRef = useRef<(i: number, param: string, value: string) => void>(
-    (i: number, param: string, value: string) => {},
-  );
+  const onChangeRowRef = useRef<
+    (i: number, param: string, value: string | boolean | undefined) => void
+  >((i: number, param: string, value: string | boolean | undefined) => {});
```

Comment:
why change this?

Reply:
It needs to support booleans, so I can now also pass the isEnabled state of the checkbox to the function.
I now could trim undefined, as I gave isEnabled a default value in the KVRowEditor.

Reply:
talking more about the `void vs {}`